flutter/example/integration_test/web_sdk_test.dart

```diff
@@ -165,7 +164,6 @@ void main() {
       testWidgets('Sentry JS SDK is not initialized', (tester) async {
```

Comment:
Do we even need this test, as we always enable by default? If not, we can also remove the `enabled` group.

Reply:
depends, this specifically tests that if `autoInitializeNativeSdks = false` then we won't init the SDK

I forgot the `autoInitializeNativeSdks = false`